src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -147,4 +169,22 @@ public static String joinPath(String ... tokens) {
     public static String getSeparator() {
         return System.getProperty("file.separator");
     }
+
+    /**
+     * Returns a {@code Byte} array given an {@code int} array. Wildcard bytes should be
+     * represented by a int with set MSB.
```

Comment:
"an integer with MSB set."

Reply:
"set bit" is a correct terminology I believe.

Reply:
"set MSB" is just the most significant bit flipped to 1 right? If that is the correct terminology, then I am okay with it.